bcda/bcdacli/cli.go

```diff
@@ -368,7 +368,7 @@ func setUpApp() *cli.App {
 					return err
 
 				}
-				if failure > 0 || skipped > 0 {
+				if failure > 0 || skipped > 0 || success == 0 {
```

Comment:
Will this cause an error / alert if no files were available for import?

Reply:
Yes, this will cause an alert if no files were successfully imported. There is [another piece of logic](https://github.com/CMSgov/bcda-app/blob/master/bcda/cclf/cclf.go#L298) that will catch an error if we've walked the contents of the directory and have no files that can be imported. I added this because when I was reviewing the logs and the output during the CCLF incident, I did see an instance of 0 files being imported.

Reply:
Per our conversation during office hours, this success statement may be a bit too aggressive; removed it!